crates/viewer/re_view_spatial/src/ui_3d.rs

```diff
@@ -116,10 +116,10 @@ impl View3DState {
         bounding_boxes: &SceneBoundingBoxes,
         space_cameras: &[SpaceCamera3D],
         scene_view_coordinates: Option<ViewCoordinates>,
+        view_ctx: &ViewContext<'_>,
+        eye_property: &ViewProperty,
     ) -> ViewEye {
-        // If the user has not interacted with the eye-camera yet, continue to
-        // interpolate to the new default eye. This gives much better robustness
-        // with scenes that change over time.
+        // Has the user interact with the eye?
```

Comment:
nothing wrong with the old comment though, no? Or did you find it factually incorrect?